docs/user-guide/managing-deployments.md

```diff
@@ -274,7 +274,7 @@ deployment "my-nginx" configured
 
 Note that `kubectl apply` attaches an annotation to the resource in order to determine the changes to the configuration since the previous invocation. When it's invoked, `kubectl apply` does a three-way diff between the previous configuration, the provided input and the current configuration of the resource, in order to determine how to modify the resource.
 
-Currently, resources are created without this annotation, so the first invocation of `kubectl apply` will fall back to a two-way diff between the provided input and the current configuration of the resource. During this first invocation, it cannot detect the deletion of properties set when the resource was created. For this reason, it will not remove them.
+Currently, resources are created without this annotation, so the first invocation of `kubectl apply` will fall back to a two-way diff between the provided input and the current configuration of the resource. During this first invocation, it cannot detect the deletion of properties set when the resource was created. For this reason, it will not remove them. So to use apply, always create resource initially with either `kubectl apply` or `kubectl create --save-config`.
```

Comment:
Since most people will skim the doc, suggest moving this to the first sentence of kubectl apply section, or add a 

**Note**: To use apply, ...

to make it more prominent.